modules/apps/ccv/types/expected_keepers.go

```diff
@@ -0,0 +1,17 @@
+package types
+
+// StakingKeeper defines the contract expected by parent-chain ccv module.
+// StakingKeeper is responsible for keeping track of latest validator set of all baby chains
+type StakingKeeper interface {
```

Comment:
I think this should be a separate module that imports the staking module. I think it's beneficial to keep the standalone and the cvv implementations in separate ctxs?

Reply:
Yes, I'm agnostic on whether this is built into x/staking or if there's a bridge module between CCV and staking.

A decision for @shahankhatch

Reply:
+1 on separate module. The staking keeper is already complex enough

Reply:
+1 to having a separate module.